docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst

```diff
@@ -10,88 +10,89 @@ While we're going to debug these issues 'manually' here, workflows (which you'll
-
-.. warning::
+.. code:: console
 
-    The contents of this configuration file will most likely only work for the virtual machine that comes with this tutorial.
-    If you are following this tutorial on your own AiiDA installation, you will have to adapt it yourself.
-    Instructions on how to do this are explained in :ref:`this appendix<appendix_computer_code_setup>`.
+    verdi computer setup --config computer.yml
 
-You can of course also manually copy and paste the content above in a file named ``computer.yml``.
-After you have created or downloaded the file to your local working environment, call the command:
+where ``computer.yml`` is a configuration file in the `YAML format <https://en.wikipedia.org/wiki/YAML#Syntax>`_)  that you can :download:`download here <include/configuration/computer.yml>`. This are its contents:
 
-.. code:: console
+.. literalinclude:: include/configuration/computer.yml
 
-    verdi computer setup --config computer.yml
+.. note::
+    When used without the ``--config`` option, ``verdi computer setup`` will prompt you for the required information, just like you have seen when :ref:`setting up a profile<setup_verdi_quicksetup>`.
+    The configuration file should work for the virtual machine that comes with this tutorial
+    but may need to be adapted when you are running AiiDA in a different environment,
+    as explained in :ref:`this appendix<appendix_computer_code_setup>`.
 
-If everything worked correctly, you should now be able to see this computer by calling:
+Finally, you need to provide AiiDA with information on how to access the ``Computer``.
+For remote computers with ``ssh`` transport, this would involve e.g. an SSH key.
+For ``local`` computers, this is just a "formality" (press enter to confirm the default cooldown time):
 
 .. code:: console
 
-    verdi computer list
+    verdi computer configure local localhost
+
 
-Before the computer can actually be used, we need to actually configure it for a specific user, which allows to connect to that computer.
-Imagine for example that you setup a remote cluster that needs to be accessed over SSH with certain credentials.
-To configure a computer, execute the command:
+Your ``localhost`` computer should now show up in ``verdi computer list``.
+Before proceeding, test that it works:
 
 .. code:: console
 
-    verdi computer configure local <COMPUTER>
+    verdi computer test localhost
 
-replacing ``<COMPUTER>`` with the name of the computer you want to configure, for example ``localhost``.
-This will interactively ask for the configuration details, which for a computer using local transport, such as the localhost, is just the one.
 
-.. note::
+Code setup
+----------
 
-    The command we use here is ``verdi computer configure local`` because the ``localhost`` was configured to use ``local`` transport.
-    If you create a computer that requires ``ssh`` transport, the command to configure connection details would be ``verdi computer configure ssh``.
```

Comment:
I can see why you would remove this detail, but given that you actually start the section with the fact that "normally" you would configure remote computers with SSH transport, I think this still needs to go in as a note. Unfortunately the syntax `verdi computer configure <TRANSPORT> <COMPUTER>` is not the clearest, due to implementation limitations

Reply:
ok, I'm adding a note back